.github/workflows/c-cpp.yml

```diff
@@ -1,22 +1,27 @@
-name: C/C++ CI
+name: OCaml bindings
 
 on: [ push, pull_request ]
 
 jobs:
   build:
     strategy:
+      fail-fat: false
       matrix:
-        cython: [cython3-legacy, cython3]
+        os:
+          - ubuntu-latest
+          - macos-latest
 
-    runs-on: ubuntu-24.04
+    runs-on: ${{ matrix.os }}
```

Comment:
What is going on here? I actually don't know what matrix.os means, or what this yaml file is at all!

Reply:
This one's a GitHub action to test the OCaml binding.  It won't be suitable for sending a PR upstream, but it's useful to test the build on ubuntu, macos (and eventually) windows. It will fail when first merged, but it has to go into the `main` branch in order to run at all.